elyra/pipeline/processor_airflow.py

```diff
@@ -73,26 +73,15 @@ def process(self, pipeline):
 
             self.log.debug("Uploading pipeline file: %s", pipeline_filepath)
 
-            github_client = Github(login_or_token=github_repo_token)
-            try:
-                # Upload to github
-                repo = github_client.get_repo(github_repo)
+            github_client = GithubClient(server_url=runtime_configuration.metadata.get('github_api_endpoint'),
```

Comment:
I need to test to confirm but I believe this won't work without https://github.com/elyra-ai/elyra/pull/1316#discussion_r582063871